platform/android/gradle/gradle-javadoc.gradle

```diff
@@ -1,18 +1,23 @@
-android.libraryVariants.all { variant ->
-    def name = variant.name
-    task "javadoc$name"(type: Javadoc) {
-        description = "Generates javadoc for build $name"
-        failOnError = false
-        destinationDir = new File(destinationDir, variant.baseName)
-        source = variant.sourceSets.collect { it.java.sourceFiles }.inject { m, i -> m + i }
-        options.windowTitle("Mapbox Maps SDK for Android $VERSION_NAME Reference")
-        options.docTitle("Mapbox Maps SDK for Android $VERSION_NAME")
-        options.header("Mapbox Maps SDK for Android $VERSION_NAME Reference")
-        options.bottom("&copy; 2015&ndash;2019 Mapbox. All rights reserved.")
-        options.links("http://docs.oracle.com/javase/7/docs/api/")
-        options.linksOffline("http://d.android.com/reference/", "$System.env.ANDROID_HOME/docs/reference")
-        options.overview("src/main/java/overview.html")
-        options.group("Mapbox Android SDK", "com.mapbox.*")
-        exclude '**/R.java', '**/BuildConfig.java'
-    }
+task javadoc(type: Javadoc) {
+    description = "Generates javadoc"
+    failOnError = false
```

Comment:
Can we enable fail on error now?

Reply:
yes, and also enabled running make android-javadoc as part of ci execution